Makefile

```diff
@@ -1,4 +1,5 @@
 PYTEST = pytest -v -s
+cleanUpList = *.pyc *.pyo *.qmlc *.jsc
```

Comment:
Please use "capital letters with underscores separating words" :pray:

Reply:
Done. If you need anything else just tell me.

Reply:
Perfect :+1: